packages/oid4vci-demo-frontend/src/types/index.ts

```diff
@@ -17,6 +17,14 @@ export interface ImageProperties {
   height?: number
 }
 
+export interface ImagePropertiesExt {
```

Comment:
the name implies it is extending, but it is not extending anything? why do we need justifyContent in the image properties? it is not really an image property

Reply:
In lack of a better name. It was ImageProperties2 before.

>should justifyContent not be moved outside of the type?
From a config perception, justifyContent relates to the logo and not the entire page, so it's in pageConfig.logo.justifyContent: 'flex-start'

We do jot have a "photo" level in the config, otherwise it should have been there. (But we should redesign the demo project at one point anyways)

Reply:
It's now LogoProperties